02week/ticTacToe.js

```diff
@@ -23,26 +30,72 @@ function printBoard() {
   console.log('2 ' + board[2].join(' | '));
 }
 
-function horizontalWin() {
-  // Your code here
+/* Run an conditional statement to make rows all equal to the same string
+In rows  make variable for function horizontalWin equal:Row "0"
+=> [[0,0], [0,1], [0,2] and Row "1" => [1,0], [1,1], [1,2]
+and Row"2 => [2,0], [2,1], [2,2]].*/
+const horizontalWin = () => {
+  if ((board[0][0] === board[0][1] && board[0][1] === board[0][2]) || (board[1][0] === board[1][1] && board[1][1] === board[1][2]) || (board[2][0] === board[2][1] &&
+      board[2][1] === board[2][2])) {
+    return true;
+  }
 }
-
-function verticalWin() {
-  // Your code here
+/* Run an conditional statement to make columns all equal to the same string
+In columns "0", "1", "2" make variable 'block' for function verticalWin equal:
+Column "0" =>[0,0], [1,0], [2,0] and Column "1" => [0,1], [1,1], ]2,0}
+Column "2" =>[0,2], [1,2],[2,2]. */
+const verticalWin = () => {
+  if ((board[0][0] === board[1][0] && board[1][0] === board[2][0]) || (board[0][1] === board[1][1] && board[1][1] === board[2][1]) || (board[0][2] === board[1][2] &&
+      board[1][2] === board[2][2])) {
+    return true;
+  }
 }
-
-function diagonalWin() {
-  // Your code here
+/* Run an conditional statement to make equal to the same string
+ make variable 'block' for function diagonalWin equal:[0,0], [1,1], [2,2] and
+ => [0,2], [1,1], ]2,0}. */
+const diagonalWin = () => {
+  if (board[0][0] === board[1][1] && board[1][1] === board[2][2] || (board[0][2] === board[1][1] && board[1][1] === board[2][0])) {
+    return true;
+  }
 }
-
-function checkForWin() {
-  // Your code here
+/* If any of the following combinations occur: playerTurn wins horizontally.
+[[0,0], [0,1], [0,2]], [[1,0], [1,1], [1,2]], [[2,0], [2,1], [2,2]] Then run
+conditional statement for horizontalWin by calling printBoard function.
+If any of the following combinations occur: playerTurn wins vertically.
+[[0,0], [1,0], [2,0]], [[0,1], [1,1], [2,1]], [[0,2], [1,2], [2,2]],Then
+continue conditional statement for verticalWin and call printBoard function.
+If any of the following combinations occur: playerTurn wins diagonally.
+[[0,0], [1,1], [2,2]], [[0,2], [1,1], [2,0]]] Then continue conditional statement
+for diagonalWin by calling printBoard function*/
+const checkForWin = () => {
+  if (horizontalWin()) {
+    printBoard();
+    return true;
+  } else if (verticalWin()) {
+    printBoard();
+    return true;
+  } else if (diagonalWin()) {
+    printBoard();
+    return true;
+  } else {
+    return false;
+  }
 }
-
+/* playerTurn starts with "X" and first chooses a row designation from range
+ of 0 to 2, then chooses a column designation from a range  of 0 to 2. */
 function ticTacToe(row, column) {
-  // Your code here
+  if (playerTurn === 'X') {
```

Comment:
Where are you calling your check for win function? And you shouldn't switch the player if that function comes back with a falsy value.